### PR TITLE
Add Portuguese Language File

### PR DESCRIPTION
**Pull request by paulofvieira**
Translated to Pt Pt

### CODE REVIEW

**Review by otacke — CHANGES_REQUESTED** [review]
Thanks a lot for contributing!

You've included all of semantics.json, however - which is not a translation file. The template would be [.en.json](https://github.com/otacke/h5p-jigsaw-puzzle/blob/master/language/.en.json). You'd have to change that.

If you don't want to run into such a mishap, you could use https://translate-h5p.tk/ which not only prevents such issues, but also has some other nice features like getting messages when there are new translations waiting, etc.

Cheers,
Oliver

File: language/pt.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        {
+          "label": "Organização do espaço",
+          "description": "Definição, em percentagem, do espaço junto ao puzzle que será utilizado para organizar as peças do mesmo."
+        },
+        {
+          "label": "Utilizar a totalidade da àrea",
+          "description": "Define se as peças devem ocupar, inicialmente, toda a àrea, inclusivê a àrea onde devem ser colocadas."
+        },
+        {
+          "label": "Possibilita o botão de \"Finalizar\" o puzzle",
```
Comment (otacke) [inline language/pt.json:62]: invalid JSON format

File: language/pt.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        },
+        {
+          "label": "Possibilita o botão de \"Tentar novamente\"",
```
Comment (otacke) [inline language/pt.json:65]: invalid JSON format

File: language/pt.json [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        },
+        {
+          "label": "Possibilita o botão para mostrar \"Solução\"",
```
Comment (otacke) [inline language/pt.json:68]: invalid JSON format

**Review by otacke — CHANGES_REQUESTED** [review]
The file that you have asked to pull now is not valid JSON. There are dangling commas where they don't belong.

I can only repeat myself: You should consider to use https://translate-h5p.tk to translate H5P content types. The issues that I now had to hint to couldn't have happened then and we'd both have had less to do.

**Comment by paulofvieira** [comment]
ty, i will do as suggested. What is the name of the project on weblate?

**Comment by otacke** [comment]
It's hosted on https://translate-h5p.tk/ as mentioned before.

**Comment by otacke** [comment]
Fixed this myself now.

**Pull request merged**